string addr_long
string addr_short
chessbot/RobCMD command